app/queries/display_ads/filtered_ads_query.rb

```diff
@@ -30,10 +32,9 @@ def call
                                 authenticated_ads(%w[all logged_out])
                               end
 
-      @filtered_display_ads = community_or_in_house_ads
+      @filtered_display_ads = type_matched_ads
```

Comment:
`cc` @Ridhwana @lboogie04 I renamed this method trying to reflect that it's also matching `external`-type ads and now dealing with `permit_adjacent_sponsors`. I'm not in love with this name, but it was the best I could think of that fit with the `_ads` pattern we have going here. I'm very open to suggestions.

Reply:
`type_of_ads` is great, I'm going with that!